### ISSUE TITLE
TWE: Create Template Issue: Update Wiki Page

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#593]
### Overview
We need to create a template issue for [TWE: Research: Wiki Page: Update (named in the format as `TWE: Role: Direct Object: Action Verb`)] so that we have a consistent, transparent, defined, and self-supporting structure for people creating them.

#### Customize this issue
 - Customize this issue by adding the Name of Issue Template. 
    - _Note: An issue template's name is different from the issue title created from the template._
    - [x] Search for `[TWE: Research: Wiki Page: Update]` by using the browser search (**Mac**: ⌘+F; **PC**: Ctrl+F)
    - [x] Replace the keyword string (including text and the brackets) below with the name of your issue template according to the template naming convention: `TWE: Role: Direct object: Action verb`
        - The role is the primary owner of the issue template (e.g., Research, Design, Product)
        - Direct object should be the item that we're updating or working on (e.g., Wiki, Interviews, Roadmap)
        - Action verb should be the main descriptor of the action items (e.g., Update, Create, Transcribe)

#### Customize the issue settings (located in right sidebar)
Change the issue settings to ensure the issue is properly categorized and easy to manage.
- [x] Under Assignees, assign yourself.
- [x] Under Projects, choose gear, choose Repository, and choose TWE Project Management.
- [x] Add the milestone: Project Management
- [x] Add a feature label specific to what the template is related to, if available.  If not clear what you should use, ask product.

#### Customize template draft
Create a draft of new template, which will be known as our "draft".
- [x] Copy resource 1.01 Content to copy for draft to a new comment on this issue.
- [x] In your draft, fill out the top part (called front matter) with the following information.  See resource 1.02 Front Matter Resources section below for definitions and examples.
    - [x] Name
    - [x] About
    - [x] Title
    - [x] Labels
- [x] Define any Dependencies (or remove the dependency section if there is no dependency)
- [x] Customize the Overview section
- [x] Fill out the Action Items section (in your draft) by defining the steps that are required to be done, keeping the following in mind:
   - [x] The person working on this issue might be new to the org, so the issue should be able to be worked on without any prior knowledge
   - [x] If the person needs to add links, or put documents in specific locations, the place where they can do that action must be linked in the resources section
      - [x] The links in the resources section provided by the issue creator should be in the same order as the Action Items 
      - [x] Refer to resource 1.03 Example Action Items for Finding Wiki Page and Template for an example of how to write clear instructions that avoid compound steps (steps within a step).
   - [x] Number each resource in the order each resource appears in the Action Items
       - [x] Group resources into numbers and decimals (e.g., Resource 1.01, Resource 2.01)
       - [x] Group the resources based on whether they are available when making the issues or they need to be created from this issue
           - If the resource is available, then group it under the **Resources for creating this issue** section (e.g., Resource 1.01, Resource 1.02, Resource 1.03)
           - If the resource will be delivered or created from this issue, then group it under the **Resource links gathered during the making of this issue** section (e.g., Resource 2.01, Resource 2.02, Resource 2.03)
- [ ] If documents or folders (sheets, docs, presentations, dashboard, etc.) will be created, find the document template that should be used by checking resource 1.04 WIKI Templates by role pages.  
  - [ ] If there is a template
     - [ ] Add a link to the template into the ` #### Resources for creating this issue`  section of your draft
  - [ ] If there is no template yet
    - [ ] Put a placeholder into the` #### Resources for creating this issue`  section of the draft `[TWE: Template Issue: [Role]: [Replace ACTION VERB][Replace DIRECT OBJECT]`
- [x] Get sign-off by Product on the draft

#### If this is for a template draft that needs testing at a later stage, please use the following handoff steps:
- [x] Add `feature: template creation (step 1)` to this issue
- [x] Once the template draft is finished, assign a team member or yourself to create testing issues 
- [x] Once the testings issues are completed, get sign-off by Product on the template draft
- [x] Create or update the template issue on the repo 
- [x] Add the link to the new issue template once created to resource 2.01.  Find by clicking on the **Get started** button to the right of the new template listed on [new issue page](https://github.com/hackforla/internship/issues/new/choose)
- [x] Update the appropriate place in the wiki for your template (see resources section)

#### Resources for creating this issue
- 1.01 Content to copy for draft
```
Draft of template issue for `TWE: Research: Wiki Page: Update`
---
**name:** TWE: Template Issue: [Replace ROLE]: [Replace ACTION VERB][Replace DIRECT OBJECT]
**about:** This issue helps you structure your template issue to be consistent with our other template issues in [Replace DIRECT OBJECT]
** title:** TWE: [Replace ROLE]: [Replace DIRECT OBJECT]: [Replace ACTION VERB]
**labels:** 'feature: board / google drive / tools, feature: missing, milestone: missing, role: missing, size: missing', 'Participant Type: missing', 'complexity: missing'
**assignees:**

---
#### Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- Customize the issue
    - [ ] Copy and paste entire template text in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
    - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
    - [ ] Search for `[Replace`
    - [ ] Replace with whatever the keyword or information is asking for.
- Customize the issue settings (located in right sidebar)
    - [ ] Under Labels, add labels for any labels identified as missing
    - [ ] Remove the corresponding missing labels
    - [ ] Under Projects, choose gear, choose "Repository," and choose TWE Project Management.
    - [ ] Add the milestone: [Replace MILESTONE]
    - [ ] Add a feature label specific to what the template is related to, if available. If not clear what you should use, ask product.
- Customize dependencies 
    - [ ] If no dependencies, delete dependency section
    - [ ] If dependencies, add the dependencies (any issue or reason that would prevent the action items from being completed)
- Remove this section.
    - [ ] Delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
We need to [Replace ACTION VERB] [Replace DIRECT OBJECT] for [Replace REASON].

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, coding, etc.

### Resources
#### Resources for creating this issue
- 1.01
- 1.02

#### Resources gathered during the completion of this issue
- 2.01
- 2.02
```

- 1.02 Front Matter Resources
   <details>
   <summary>Front Matter Fields and Meaning</summary>

   - name: (name that appear on the new issue tab)
   - about: (is the description that appears next to the issue template on the new issue tab)
   - title: (the title that appears when the issue is created.  User then customizes this title for their issue)
       - Use the naming convention: TWE: Role: Direct object: Action verb
       - If more than one word should go into the direct object spot, use multiple square brackets with the keyword (see example below)
   - labels: (choose the labels that will be the same each time, and choose missing labels for the ones that depend on content)
   - assignees:  
   </details>
   <details>
   <summary>Front Matter Field Example</summary>

   - name: 'TWE: Template Issue: Research: Create Roadmap'
   - about:  This issue helps you structure your template issue to be consistent with our other template issues in roadmap
   - title: 'TWE: Research: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF RESEARCH METHOD]: Roadmap : Create'
   - labels: 'feature: research', 'feature: roadmap', 'milestone: missing', 'role: UI/UX research', 'Participant Type: missing', 'Research 000: missing', 'complexity: missing'
   - 'size: 0.50pt'
   - assignees: ''
   </details>

- 1.03 Example Action Items for Finding Wiki Page and Template
   <details>
   <summary>Example Action Items</summary>

   ```
   ### Action Items
   - Customize wiki page
        - [ ] Go to link 1.01 Internship Wiki, Join the team page
        - [ ] Find wiki page for this issue by looking at role pages.
        - [ ] Copy the link for the wiki page.
        - [ ] Add the link to Resource 2.01
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   - Customize document template.
       - [ ] Go to link 1.02 Design Process Overview
       - [ ] Find document template that should be used in this issue.
       - [ ] Copy the link for the template.
       - [ ] Add the link to Resource 2.02
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   ```

   </details>

   <details>
   <summary>Example Resource Lists</summary>

   ```
   ### Resources for creating this issue
   - 1.01. [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team)
   - 1.02. [Design Process Overview](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

   ### Resource links gathered during the making of this issue
   - 2.01. [Wiki page for [Replace TYPE OF ROLE]]
   - 2.02. [TWE: Template: [Replace NAME OF TEMPLATE]]
   ```

   </details>

- 1.04 WIKI Templates by role pages
   - [TWE: WIKI: UX Research Guides Templates](https://github.com/hackforla/internship/wiki/UX-Research-Guides-Templates) 
   - [Design](https://github.com/hackforla/internship/wiki/Design-Process-Overview)
   - [Product](https://github.com/hackforla/internship/wiki/Product)
   - [Business Analyst / Data Science]
   - [Software Engineer]
   - [Marketing]
   - default document templates, in case you don't have one yet, can be found on the [Design Process Overview page](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

#### Resources gathered during the completion of this issue
- 2.01 [TWE: Research: [Replace TITLE of RESEARCH]: Update Wiki Page](https://github.com/hackforla/internship/issues/new?template=research--update-wiki-page.md)

**Comment by pandanista** [comment]
## Revised Update Wiki Page Template

---
**name:** TWE: Template Issue: Research: Update Wiki Page
**about:** This issue helps you structure your template issue to be consistent with our other template issues in research wiki page updates
**title:** TWE: Research: [Replace TITLE of RESEARCH]: Update Wiki Page
**labels:** 'feature: research audits', 'p-feature: wiki', 'Participant Type: missing', 'Research 000: missing', 'size: 1pt', 'role: UI/UX research', 'milestone: missing'
**assignees:**

---

### Overview
We need to review and update the wiki page for [Replace TITLE of RESEARCH] so that we can stay organized and consistent across all team members and working times, and future TWE volunteers can replicate our processes.

<b>Resources/Instructions </b>: This section is at the bottom of this issue (scroll to the bottom to view it now). You will be asked to add links to this section while completing the issue.

Tip: Use two windows side by side. One with the issue open and the other one with resource links displayed to avoid back and forth. To prevent loss of work, refresh both windows after each edit.

### Action Items

#### Update Issue Labels (located in the right sidebar)
- [ ] Under Assignees, assign yourself  
- [ ] Under Labels, click the gear-shaped setting icon (![Screenshot 2023-01-11 at 10 50 11 AM](https://user-images.githubusercontent.com/57029070/211904252-46521281-b8f8-495d-b31f-d06e425f0096.png)), then apply the label `Participant Type: [Replace TYPE of PARTICIPANT]`
- [ ] Remove the `Participant Type: missing` label
- [ ] Under Projects, click the setting icon, then choose `Repository`, and choose `Program Management`
- [ ] Add the milestone: click the setting icon, then choose `06.01.02 Research Audit and Update Wiki Page` 
- [ ] Remove the label `milestone: missing`

#### Review Google Drive
- [ ] Go to the Shared Drives > Internship > Internships > Research > Research by Plan # and Name (Resources # 1.01)
- [ ] Go to the research plan folder of ```RP[Replace 000]```
- [ ] Review all files in the folder of ```RP[Replace 000]```
- [ ] Find all updated files that will be reflected in the wiki page update, see the action items below

#### Find Wiki Page
- [ ] Go to the wiki page: Research Output Overview (Resources # 1.02)
- [ ] Scroll down to the section ```Research by Plan Number```  
- [ ] Open the hyperlink ```Research Plans, sorted by number``` (Resources # 1.03)
- [ ] Locate the Research Plan you attempt to update the wiki page, often titled as ```Research Plan [Replace 000]: [Replace TITLE of RESEARCH]```
- [ ] Open the wiki page of `Research Plan [Replace 000]`
- [ ] Copy the URL of the wiki page of `Research Plan [Replace 000]`
- [ ] Paste the URL at the end of Resources # 2.01 in `()` so it becomes a hyperlink  

#### Update Wiki Page (Resources # 2.01)
- Open the link in Resources # 2.01 if it is not open
- Initiate an update on wiki page
    - [ ] Choose "Edit" in the title section
- Update research plan agenda on wiki page
    - [ ] Update ```Proposed Research Agenda``` 
         - [ ] Update date information for  ```Start```, ```Data Collected``` and ```End```
- Update research plan details and status on wiki page    
    - [ ] Update ```Summary of Research```
        - [ ] Update ```Question```
        - [ ] Update ```Purpose``` 
        - [ ] Update ```Methods```
    - [ ] Update ```Current Status```
 - Check all issues related to the research plan 
    - [ ] Check issue labels
         - [ ] Go to the repository [hackforla/intership](https://github.com/hackforla/internship) 
         - [ ] Click "Issues" OR Use this [direct link](https://github.com/hackforla/internship/issues)
         - [ ] Go over all issues, either open or closed, related to ```RP[Replace 000]```
         - [ ] Check the consistency in the language describing the research plan (e.g., "intake" vs. "onboarding")
              - [ ] Correct any inconsistency in description language
         - [ ] Check if all issues related to```RP[Replace 000]``` have the correct label ```Research: RP[Replace 000]```
              - [ ] Add the correct label if the research label is missing
              - [ ] Correct any incorrect research label
    - [ ] Check ```All Issues``` 
         - [ ] Go back to the wiki page
         - [ ] Open the hyperlink of ```All Issues```
         - [ ] Check if every issue listed, either open or closed, is associated with a label ```RP[Replace 000]```, which indicates the wiki page correctly identifies all issues related to this research plan
- Update research outstanding tasks and assets on wiki page
     - [ ] Update ```Outstanding Task Items``` 
     - [ ] Update ```Assets```
         - [ ] Update the second link under ```Google Drive Folders```
             - [ ] Go to the Shared Drives > Internship > Internships > Research > Research by Plan # and Name (Resources # 1.01)
              - [ ] Find the research folder of ```RP[Replace 000]```
              - [ ] Copy the sharable link of ```RP[Replace 000]``` folder
              - [ ] Update the link for ```RP folder```
          - [ ] Update the links under ```Research Plan```
              - [ ] Go to the Shared Drives > Internship > Internships > Research > Research by Plan # and Name (Resources # 1.01) > ```RP[Replace 000]```
              - [ ] Find the updated research plan document, often titled as ```TWE: IS[Replace 00]:RP[Replace 000]:[Replace the TITLE of Reserach] Research Plan```
              - [ ] Copy the sharable link of the updated research plan document
              - [ ] Update the link for ```Research Plan```
          - [ ] Update the links under ```Scripts Docs```
              - [ ] Go to the Shared Drives > Internship > Internships > Research > Research by Plan # and Name (Resources # 1.01) > ```RP[Replace 000]```
              - [ ] Find all updated script documents, often titled as ```TWE:IS[Replace 00]:RP[Replace 000]:[Replace TITLE of RESEARCH] Script/Form```
              - [ ] Copy the sharable link(s) of the updated script documents
              - [ ] Update the link(s) for ```Scripts Docs```
              - [ ] Leave it blank if there is no update in the Google Drive
          - [ ] Update  the links under ```Interview Recordings & Transcripts``` (for qualitative data) or ```Survey``` (for quantitative data)
              - [ ] Go to the Shared Drives > Internship > Internships > Research > Research by Plan # and Name (Resources # 1.01) > ```RP[Replace 000]```
              - [ ] Find all updated interview recordings & transcripts OR survey documents
              - [ ] Copy the sharable link(s) of all updated interview recordings & transcript OR survey documents
              - [ ] Update the link(s) for ```Interview Recordings & Transcripts``` OR ```Survey```
              - [ ] Leave it blank if there is no update in the Google Drive
          - [ ] Update the links under ```Synthesis``` 
              - [ ] Go to the Shared Drives > Internship > Internships > Research > Research by Plan # and Name (Resources # 1.01) > ```RP[Replace 000]```
              - [ ] Find the updated synthesis documents, often titled as ```TWE:IS[Replace 00]:RP[Replace 000]:[Replace TITLE of RESEARCH] Synthesis```
              - [ ] Copy the sharable link of the updated synthesis document
              - [ ] Update the link for ```Synthesis```              
              - [ ] Leave it blank if there is no update in the Google Drive
          - [ ] Update the links under ```Presentation of Findings``` 
             - [ ] Go to the Shared Drives > Internship > Internships > Research > Research by Plan # and Name (Resources # 1.01) > ```RP[Replace 000]```
              - [ ] Find the updated presentation documents, often titled as ```TWE:IS[Replace 00]:RP[Replace 000]:[Replace TITLE of RESEARCH] Presentation```
              - [ ] Copy the sharable link of the updated presentation document
              - [ ] Update the link for ```Presentation of Findings```              
              - [ ] Leave it blank if there is no update in the Google Drive
          - [ ] Update the links under ```Action Items Spreadsheet``` 
              - [ ] Go to the Shared Drives > Internship > Internships > Research > Research by Plan # and Name (Resources # 1.01) > ```RP[Replace 000]```
              - [ ] Find the updated action item spreadsheet, often titled as ```TWE:IS[Replace 00]:RP[Replace 000]Action Steps:[Replace TITLE of RESEARCH]```
              - [ ] Copy the sharable link of the updated action item spreadsheet
              - [ ] Leave it blank if there is no update in the Google Drive
- Update related wiki pages 
    - [ ] Update ```Related Wiki Pages```
         - [ ] Copy the URL of Research Home Page (Resources # 1.04)
         - [ ] Paste behind the Research Home Page in `()` so it becomes a hyperlink
         - [ ] Copy the URL of Research Output Overview (Resources # 1.02)         
         - [ ] Paste behind the Research Output Overview in `()` so it becomes a hyperlink
- [ ] Ask Research Lead to Review the Updated Wiki Page
- [ ] Ask Product Team to Review the Updated Wiki Page

### Resources/Instructions

#### Resources for creating this issue
1.01 [Google Drive, Research Folder](https://drive.google.com/drive/folders/1LLHYcQCRmpC3DX_ZMj4BXNCmb_XliUXd?usp=sharing)
1.02 [Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
1.03 [Research: Plans: by Number](https://github.com/hackforla/internship/wiki/Research:-Plans:-by-Number)
1.04 [Research Home Page](https://github.com/hackforla/internship/wiki/Research)

 #### Resources gathered during the completion of this issue
2.01 [RP[Replace 000]: Wiki Page]

**Comment by pandanista** [comment]
Need to update the template further by adding `customize the issue` and `customize the settings` sections, then we will update the order of the resources if needed. 

See [this comment](https://github.com/hackforla/internship/issues/534#issuecomment-2127648344) for more info.

**Comment by pandanista** [comment]
@jinyan0425 Would you mind finish updating [this wiki page update template](https://github.com/hackforla/internship/issues/593#issuecomment-2136411580)? Thank you. 
- The customize the issue and customize the settings can be found in [this comment](https://github.com/hackforla/internship/issues/533#issuecomment-2144027444) you drafted in #533.

**Comment by jinyan0425** [comment]
> @jinyan0425 Would you mind finish updating [this wiki page update template](https://github.com/hackforla/internship/issues/593#issuecomment-2136411580)? Thank you.
> 
> * The customize the issue and customize the settings can be found in [this comment](https://github.com/hackforla/internship/issues/533#issuecomment-2144027444) you drafted in [TWE: Revise Research: Research Plan Template #533](https://github.com/hackforla/internship/issues/533).

@pandanista I have added the customization section.

**Comment by pandanista** [comment]
For the update wiki page template, there is an action item that asks the assignee to link the issues related to the RP### label. I'd like to get some clarification on the scope of the action item. 
- Does it only ask the assignee to use the [Feature: RP008](https://github.com/hackforla/internship/issues?q=is%3Aissue+label%3A%22Research%3A+RP008%22) to check and see if they are all labelled correctly using the RP###? 
- Does the assignee check all open and closed issues using [Internship issues direct link](https://github.com/hackforla/internship/issues) to check all the issues?

**Comment by ExperimentsInHonesty** [comment]
@jinyan0425 Next step is to test this issue using this [testing template](https://github.com/hackforla/internship/issues/new?assignees=&labels=feature%3A+missing%2C+feature%3A+research%2C+feature%3A+test+a+template+%28step+2a%29%2C+good+first+issue%2C+milestone%3A+missing%2C+Participant+Type%3A+TWE+Team+Member%2C+role%3A+UI%2FUX+research%2C+size%3A+3pt%2C+time+sensitive&projects=&template=research--test--issue-template.md&title=Research%3A+Test%3A+Issue+Template%3A+%5BReplace+with+TEMPLATE+NAME%5D) Once you have created the testing issue, add the link to a new comment here and the label `status: template testing (step 2b)` to this issue.

**Comment by pandanista** [comment]
I updated the[testing template](https://github.com/hackforla/internship/issues/new?assignees=&labels=feature%3A+missing%2C+feature%3A+research%2C+feature%3A+test+a+template+%28step+2a%29%2C+good+first+issue%2C+milestone%3A+missing%2C+Participant+Type%3A+TWE+Team+Member%2C+role%3A+UI%2FUX+research%2C+size%3A+3pt%2C+time+sensitive&projects=&template=research--test--issue-template.md&title=Research%3A+Test%3A+Issue+Template%3A+%5BReplace+with+TEMPLATE+NAME%5D), and some of the changes include: 
- Correcting `feature: test a template (step 2a)` to `feature: template test (step 3)` since the step 3 label needs to be added to any new testing issues assignees create instead of step 2a;
- Adding `status: template testing (step 2b)` to the create template draft issue in the instructions.

@jinyan0425 Let me know if we need to make more changes to the [testing template](https://github.com/hackforla/internship/issues/new?assignees=&labels=feature%3A+missing%2C+feature%3A+research%2C+feature%3A+test+a+template+%28step+2a%29%2C+good+first+issue%2C+milestone%3A+missing%2C+Participant+Type%3A+TWE+Team+Member%2C+role%3A+UI%2FUX+research%2C+size%3A+3pt%2C+time+sensitive&projects=&template=research--test--issue-template.md&title=Research%3A+Test%3A+Issue+Template%3A+%5BReplace+with+TEMPLATE+NAME%5D) while you set it up. Thank you.

**Comment by jinyan0425** [comment]
Shall we move this to the ICEBOX? @pandanista  

Could you please check this with Bonnie next week? Thank you!

**Comment by ExperimentsInHonesty** [comment]
- This issue is waiting on #https://github.com/hackforla/internship/issues/614 to be completed

**Comment by ExperimentsInHonesty** [comment]
I think in the template it would be hard to do this:
Add the correct label if the research plan number label is missing
because the instructions that proceed it, tell you to look at the issues that have the label.  
So if we want people to find any research issues that are applicable that don't have the label, we should have them perform a search of the research plan name possibly?

**Comment by ExperimentsInHonesty** [comment]
This is ready for testing.  Please use it on the research plan RP023

**Comment by jinyan0425** [comment]
@pandanista Ying, let us discuss this issue next Tuesday. Please include this in the meeting agenda. Thanks!

**Comment by jinyan0425** [comment]
I will update this template accordingly based on our meeting discussion.

**Comment by jinyan0425** [comment]
<img width="750" alt="Image" src="https://github.com/user-attachments/assets/2cc75341-1766-4814-b46d-eef7e00d98d1" />

@pandanista If you're good with this update, we can go ahead assigning Neil to test it with RP025 once he completes his current issue.

**Comment by pandanista** [comment]
- The update looks good. Thank you, @jinyan0425.
- Once step 3 of the template testing gets created, we can add the testing issue number in the comment above if needed.
- Leaving this issue in the review column so we can revisit the template and incorporate any changes needed during the testing phase.

**Comment by jinyan0425** [comment]
> * The update looks good. Thank you, [@jinyan0425](https://github.com/jinyan0425).
> * Once step 3 of the template testing gets created, we can add the testing issue number in the comment above if needed.
> * Leaving this issue in the review column so we can revisit the template and incorporate any changes needed during the testing phase.

@pandanista Would you like me to create the template testing (step 3)? Sorry that I realized that #611 is just step 2b.

**Comment by pandanista** [comment]
@jinyan0425 No. The tester/assignee for step 2a, which in this case is #611, is the one that will set up step 3. So whenever Neil starts to test, we can add the issue he will be creating in this issue to make sure we are tracking it.

**Comment by jinyan0425** [comment]
> [@jinyan0425](https://github.com/jinyan0425) No. The tester/assignee for step 2a, which in this case is [#611](https://github.com/hackforla/internship/issues/611), is the one that will set up step 3. So whenever Neil starts to test, we can add the issue he will be creating in this issue to make sure we are tracking it.

You are right! I haven't tested anything for a long time! Let us wait for him to finish his current issue first. @pandanista
